extensions-modules/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "Shared modules for Carbon extensions",
   "dependencies": {
-    "dataprotocol-client": "git://github.com/Microsoft/sqlops-dataprotocolclient.git#release",
+    "dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#0.1.2",
```

Comment:
so we'll need to bump this every time we merge DMP changes?

Reply:
yeah, similar to sqltoolsservice. We could just patch a tag, but we have had problems with hashes being cache and all sorts of weird things happening.